### PR TITLE
i18n(es): Update astro components

### PR DESCRIPTION
**Pull request by glopzel**
#### What kind of changes does this PR include?

- New or updated content
- Changes to the docs site code

#### Description

- What does this PR change? delete reference of exporting Props when using TypeScript + small change on typescript.md (everything else seems to be updated on that file)

`hackoberfest-accepted`

### CODE REVIEW

File: src/pages/es/core-concepts/astro-components.md [hunk 1]
```diff
@@ -277,12 +277,12 @@ const name = "Astro"
 ```
 
 
-También puedes definir props con TypeScript exportando una interfaz de tipo `Props`. Astro recogerá automáticamente cualquier interfaz `Props` exportada y dará advertencias/errores de tipo para su proyecto. A estos accesorios también se les pueden dar valores predeterminados cuando se desestructuran desde `Astro.props`
+También puedes definir props con TypeScript usando una interfaz de tipo `Props`. Astro recogerá automáticamente la interfaz `Props` en el frontmatter y dará advertencias/errores de tipo para su proyecto. A estos accesorios también se les pueden dar valores predeterminados cuando se desestructuran desde `Astro.props`.
```
Comment (fredoist) [inline src/pages/es/core-concepts/astro-components.md:280]:
```suggestion
También puedes definir props con TypeScript usando una interfaz de tipo `Props`. Astro recogerá automáticamente la interfaz `Props` en el frontmatter y dará advertencias/errores de tipo para tu proyecto. A estas propiedades también se les puede dar valores predeterminados cuando se desestructuran desde `Astro.props`.
```

**Review by fredoist — COMMENTED** [review]
Great work Angela, I left a suggestion for the second paragraph.

**Review by agustinmulet — APPROVED** [review]
LGTM! 🙌

**Review by delucis — APPROVED** [review]
Thanks @glopzel! Just added the label 🙌

@fredoist & @agustinmulet feel free to open PRs for Hacktoberfest review credit if you want! (More details in [our Hacktoberfest guide](https://github.com/withastro/docs/blob/main/.github/hacktoberfest.md#recognizing-i18n-pr-reviewers).)

**Pull request merged**